src/main/java/betterquesting/api/utils/NBTConverter.java

```diff
@@ -5,15 +5,77 @@
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonPrimitive;
+import com.google.gson.stream.JsonWriter;
 import net.minecraft.nbt.*;
 import org.apache.logging.log4j.Level;
 
+import java.io.IOException;
+import java.util.*;
 import java.util.Map.Entry;
-import java.util.Set;
-import java.util.TreeSet;
 
 public class NBTConverter
 {
+    /**
+     * Convert NBT tags to a JSON object
+     */
+    private static void NBTtoJSON_Base(NBTBase value, boolean format, JsonWriter out) throws IOException {
```

Comment:
Hm, why add this and keep the existing NBTtoJSON_Base and the various mthods that call the original and not this method